doc/corewar-glossary.html

```
@@ -0,0 +1,675 @@
+              mov   example, &lt;example
+</pre>
+
+Modern incendiary bombs use separate <code class="code94">SPL</code> /
+<code class="code94">MOV</code> bombs:
+
+<pre class="code94">
+    <span class="label">sp</span>        spl   0, sm+1            <span class="comment">; spl half of the incendiary</span>
+
+              ...
+
+    <span class="label">sm</span>        mov   sp, >sp            <span class="comment">; mov half of the incendiary</span>
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ > ].

(spec-char-escape)